daqPinDefs.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * @brief Definitions and constants for the DAQ wheel board. Initializes with a wheel speed sensor pin, 
+ * brake temp sensor pin, sus pos sensor pin, and can frame address.
+ * 
+ */
+class WheelBoard
+{
+    public:
+        int wheel_speed_sensor_pin;
+        int brake_temp_sensor_pin;
+        int suspension_position_sensor_pin;
+        int can_frame_address;
+        float brake_temp_scalar;
+        float brake_temp_offset;
+        float suspension_position_scalar;
+        float suspension_position_offset;
+
+        WheelBoard(int wss_pin, int bts_pin, int sps_pin, CANFrameAddress can_frame)
+        {
+            wheel_speed_sensor_pin = wss_pin;
+            brake_temp_sensor_pin = bts_pin;
+            suspension_position_sensor_pin = sps_pin;
+            can_frame_address = can_frame;
+            brake_temp_scalar = 1;
+            brake_temp_offset = 0;
+            suspension_position_scalar = 1;
+            suspension_position_offset = 0;
+        }
+};
```

Comment:
I would recommend having the pin definitions and member variables all be private, and have functions like ReadWheelSpeedSensorDuration() and GetWheelSpeed() as public functions. You can also hard-code the pin values as constexpr (which should be kCamelCase according to the Google style guide) where you declare them as private variables, since the board is board-specific and therefore doesn't need to be compatible with other pin mappings. I think the convention in C++ is that you should avoid public member variables and #defines.